cfme/common/vm.py

```diff
@@ -792,7 +793,15 @@ def create_on_provider(self, timeout=900, find_in_cfme=False, delete_on_failure=
             find_in_cfme: Verifies that VM exists in CFME UI
             delete_on_failure: Attempts to remove VM on UI navigation failure
         """
-        vm = deploy_template(self.provider.key, self.name, self.template_name, **kwargs)
+        try:
+            from polarion_docstrings.parser import DocstringParser
+            assignee = DocstringParser(request.function.__doc__).parse()["assignee"].value
```

Comment:
`request` defaults to `None`, and I have seen many usages of `create_on_provider` that do not pass request. I think this should be checked.

Reply:
Oh I see, you're catching the `TypeError` from request being `None`

Reply:
I think this `try/except` block should be changed to a `if request:` condition. What do you think?